src/core/sagas/addons.js

```diff
@@ -7,9 +7,8 @@ import { hideLoading, showLoading } from 'react-redux-loading-bar';
 import { call, put, select, takeEvery } from 'redux-saga/effects';
 /* eslint-enable import/order */
 
-import { loadEntities } from 'core/actions';
 import { fetchAddon as fetchAddonFromApi } from 'core/api';
-import { FETCH_ADDON } from 'core/reducers/addons';
+import { loadAddons, FETCH_ADDON } from 'core/reducers/addons';
```

Comment:
Nitpick but I think we usually load constants first. It technically is alphabetical that way but I guess until we have a rule for sorting imports... 😞